src/selectors/rankThoughtsFirstMatch.js

```diff
@@ -74,7 +74,8 @@ export default (state, pathUnranked) => {
       value,
       // NOTE: we cannot throw an error if there is no parent, as it may be a floating context
       // unfortunately this that there is no protection against a (incorrectly) missing parent
-      rank: parent ? parent.rank : 0
+      rank: parent ? parent.rank : 0,
+      uuid: parent ? parent.uuid : ''
```

Comment:
Let’s set it to `null` if there is no uuid. Thats more explicit than an empty string.

Reply:
Sure!